lametro/templates/lametro/index.html

```diff
@@ -66,7 +66,37 @@ <h2>
                         {% include "partials/meeting_details_next.html" %}
                     </div>
                     <div class='col-md-5'>
-                        {% include "partials/index_metro_description.html" %}
+                        {% if todays_meetings %}
```

Comment:
Nesting the display logic like this will result in today's meetings not being shown when there is a current meeting. Can you revise the conditional logic in the index template so today's meetings are always shown if they exist?